tests/mgbench/benchmark.py

```diff
@@ -198,87 +242,129 @@ def filter_benchmarks(generators, patterns):
-                              should_execute))
-                    # We don't have to execute the next iteration when
-                    # `should_execute` becomes the same order of magnitude as
-                    # `count * 10`.
-                    if should_execute / (count * 10) < 10:
-                        count = should_execute
-                        break
-                    else:
-                        count = count * 10
-                memgraph.stop()
-                config.set_value(*config_key, value={
-                    "count": count,
-                    "duration": args.single_threaded_runtime_sec})
-            else:
-                print("Using cached query count of", cached_count["count"],
-                      "queries for", cached_count["duration"],
-                      "seconds of single-threaded runtime.")
-                count = int(cached_count["count"] *
-                            args.single_threaded_runtime_sec /
-                            cached_count["duration"])
+    for with_fine_grained_authorization in [False, True]:
+        if with_fine_grained_authorization:
+            memgraph.start_preparation()
+            client.execute(file_path=dataset.get_file(), num_workers=args.num_workers_for_import)
+            client.execute(
+                queries=[
+                    ("CREATE USER user IDENTIFIED BY 'test';", {}),
+                    ("GRANT ALL PRIVILEGES TO user;", {}),
+                    ("GRANT CREATE_DELETE ON EDGE_TYPES * TO user;", {}),
+                    ("GRANT CREATE_DELETE ON LABELS * TO user;", {}),
+                ]
+            )
+            client = runners.Client(
+                args.client_binary,
+                args.temporary_directory,
+                args.bolt_port,
+                username="user",
+                password="test",
+            )
+            memgraph.stop()
+
+        test_type = (
+            WITH_FINE_GRAINED_AUTHORIZATION if with_fine_grained_authorization else WITHOUT_FINE_GRAINED_AUTHORIZATION
+        )
 
-            # Benchmark run.
-            print("Sample query:", get_queries(func, 1)[0][0])
-            print("Executing benchmark with", count, "queries that should "
-                  "yield a single-threaded runtime of",
-                  args.single_threaded_runtime_sec, "seconds.")
-            print("Queries are executed using", args.num_workers_for_benchmark,
-                  "concurrent clients.")
-            memgraph.start_benchmark()
-            ret = client.execute(queries=get_queries(func, count),
-                                 num_workers=args.num_workers_for_benchmark)[0]
-            usage = memgraph.stop()
-            ret["database"] = usage
+        for group in sorted(tests.keys()):
+            for test, funcname in tests[group]:
+                test = f"{test} {test_type}"
+                log.info("Running test:", "{}/{}".format(group, test))
```

Comment:
change this line to `log.info("Running test:", "{}/{}/{}".format(group, test, test_type))`